tests/test_extensions/test_ext_intersphinx.py

```diff
@@ -551,30 +552,32 @@ def test_intersphinx_role(app, warning):
 
     app.build()
     content = (app.outdir / 'index.html').read_text(encoding='utf8')
-    wStr = warning.getvalue()
+    warnings = strip_colors(warning.getvalue()).replace(str(app.srcdir), "src").replace('\\', '/').splitlines()
```

Comment:
I wouln't replace `str(app.srcdir)` but rather use f-strings when checking the output. That way you don't have unexpected surprises with path separators or escaped characters.

Reply:
done